package/ios/CameraView.swift

```diff
@@ -45,6 +45,14 @@ public final class CameraView: UIView, CameraSessionDelegate {
   // other props
   @objc var isActive = false
   @objc var torch = "off"
+  @objc var torchLevel: NSNumber = 0.7
+  @objc var torchDelay: NSNumber = 0
+  @objc var torchDuration: NSNumber = 0
+  @objc var backgroundLevel: NSNumber = 0.0
+  @objc var backgroundDelay: NSNumber = 0
+  @objc var backgroundDuration: NSNumber = 0
+  @objc var enableBackgroundTorch = false
+  @objc var isTorchOn = false
```

Comment:
Is this default going to interfere with normal camera operation?

Reply:
Shouldn't do